vast/integration/vast_integration_suite.yaml

```diff
@@ -283,14 +283,34 @@ tests:
       - command: -N export ascii
         input: queries/multi_addr.txt
 
+  Pivot with PCAP:
+    tags: [node, pivot, pcap]
```

Comment:
This should just be named `Pivot`, and the `pcap` tag should be removed.

Reply:
Done.